### ISSUE TITLE
Can lib and example be separate?

### ISSUE THREAD

**Issue by andresn** [paramaggarwal/react-native-device-motion#2]
Hi @paramaggarwal,

When npm installing, it will pull in the nice example you have, but will also install a redundant react-native dependency within a project. Therefore, I think this project and its users would benefit from being organized more like this one: https://github.com/pwmckenna/react-native-motion-manager. Including matching its installation steps since those seem to be standard across popular RN components.

Thank you in advance for your time!

**Comment by paramaggarwal** [comment]
Yes, good point. I am not actively using this library - so it would be great if we could have a PR raised - or better, maybe a new home for the project managed by someone actively using it.
